pkg/kubernetes/pod/v1alpha1/kubernetes.go

```diff
@@ -98,10 +118,28 @@ func (k *kubeclient) getClientOrCached() (*clientset.Clientset, error) {
 
 // List returns a list of pod
 // instances present in kubernetes cluster
-func (k *kubeclient) List(opts metav1.ListOptions) (*v1.PodList, error) {
+func (k *Kubeclient) List(opts metav1.ListOptions) (*v1.PodList, error) {
 	cli, err := k.getClientOrCached()
 	if err != nil {
 		return nil, err
 	}
 	return k.list(cli, k.namespace, opts)
 }
+
+// Delete deletes a pod instance present in kubernetes cluster
+func (k *Kubeclient) Delete(name string, opts *metav1.DeleteOptions) error {
+	cli, err := k.getClientOrCached()
+	if err != nil {
+		return err
+	}
+	return k.del(cli, k.namespace, name, opts)
+}
+
+// Get gets a pod object present in kubernetes cluster
+func (k *Kubeclient) Get(name string, opts metav1.GetOptions) (*v1.Pod, error) {
+	cli, err := k.getClientOrCached()
```

Comment:
can have  check for name is empty or not.

Reply:
Done